blocks/core/ff_container/ff_container-expandable/ff_container-expandable.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+$(function(){ 
+	$("[data-ff-action='expandable']").click(function(){
+		var $expandableContent = $("[data-ff='expandable-content']"),
+		$expandableIcon = $("[data-icon='expandable-icon']"),
+		$expandableText = $("[data-ff='expandable-text']");
+		
+		var $expandableOpenText = $expandableText.attr("data-open-text"),
+		$expandableClosedText = $expandableText.attr("data-closed-text");
+
+		var isVisible = $expandableContent.is(":visible");
+		var newText = isVisible ? $expandableClosedText : $expandableOpenText;
+
+		if (isVisible) {
+			$expandableContent.hide();
+			$expandableIcon.removeClass("ff_icon-page-up-open-blue").addClass("ff_icon-page-down-open-blue");
```

Comment:
Why does the class need to specifically reference the colour? Can't you set it to something like `ff_icon-page-up-open` and let the theme pick the colour through CSS?